tests/unit/keras_app/test_views.py

```diff
@@ -35,17 +38,40 @@ def setUp(self):
 
     def test_keras_import(self):
         # Test 1
-        sample_file = open(os.path.join(settings.BASE_DIR, 'example/keras', 'vgg16.json'), 'r')
-        response = self.client.post(reverse('keras-import'), {'file': sample_file})
+        sample_file = open(os.path.join(settings.BASE_DIR,
+                                        'example/keras',
+                                        'vgg16.json'), 'r')
+        response = self.client.post(reverse('keras-import'),
+                                    {'file': sample_file})
         response = json.loads(response.content)
         self.assertEqual(response['result'], 'success')
         # Test 2
-        sample_file = open(os.path.join(settings.BASE_DIR, 'example/caffe', 'GoogleNet.prototxt'), 'r')
-        response = self.client.post(reverse('keras-import'), {'file': sample_file})
+        sample_file = open(os.path.join(settings.BASE_DIR,
+                                        'example/caffe',
+                                        'GoogleNet.prototxt'), 'r')
+        response = self.client.post(reverse('keras-import'),
+                                    {'file': sample_file})
         response = json.loads(response.content)
         self.assertEqual(response['result'], 'error')
         self.assertEqual(response['error'], 'Invalid JSON')
 
+    def test_keras_import_sample_id(self):
+        response = self.client.post(
+            reverse('keras-import'),
+            {'sample_id': 'vgg16'})
+        response = json.loads(response.content)
+        self.assertEqual(response['result'], 'success')
+        self.assertEqual(response['net_name'], 'vgg16')
+        self.assertTrue('net' in response)
+
+    def test_keras_import_invalid_sample_id(self):
```

Comment:
Either split the Caffe `sample_id` tests into two or combine these into one for uniformity